.github/workflows/test.yml

```
@@ -0,0 +1,62 @@
+    
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@v4
+      
+    - name: Setup Rust
+      uses: dtolnay/rust-toolchain@stable
+      with:
+        toolchain: stable
+        
+    - name: Cache Cargo
+      uses: actions/cache@v3
```

actionlint (1.7.8)

the runner of "actions/cache@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)